nilearn/plotting/displays.py

```diff
@@ -859,7 +908,8 @@ def add_markers(self, marker_coords, marker_color='r', marker_size=30,
             display_ax.ax.scatter(xdata, ydata, s=marker_size,
                                   c=marker_color, **kwargs)
 
-    def annotate(self, left_right=True, positions=True, size=12, **kwargs):
+    def annotate(self, left_right=True, positions=True, scalebar=False,
+                 size=12, **kwargs):
```

Comment:
The "scalebar" argument is not documented.

The documentation of the "scalebar" parameter should point to the "draw_scale_bar" for finer control.

Reply:
Done